bugbug/repository.py

```diff
@@ -465,12 +518,36 @@ def download_commits(repo_dir, date_from):
 
                     files_touched_90_days[copied] = files_touched_90_days[orig]
 
+                    orig_directories = [
+                        directory
+                        for directory in os.path.dirname(orig).split("/")[:2]
+                        if directory != ""
+                    ]
+                    copied_directories = [
+                        directory
+                        for directory in os.path.dirname(copied).split("/")[:2]
```

Comment:
Maybe the `os.path.dirname(path).split("/")[:2] if directory != ""` could be extracted somewhere?

Reply:
Also this will be at most a list of size 2, right?

Reply:
> Maybe the `os.path.dirname(path).split("/")[:2] if directory != ""` could be extracted somewhere?

But for the original and copied file paths, we'd still have to do it twice as the path may change as well. 

> Also this will be at most a list of size 2, right?

Yes.